actors/miner/src/lib.rs

```diff
@@ -2002,56 +1937,26 @@ impl Actor {
             })?
             .ok_or_else(|| actor_error!(not_found, "no pre-commited sector {}", sector_number))?;
 
-        let max_proof_size = precommit.info.seal_proof.proof_size().map_err(|e| {
-            actor_error!(
-                illegal_state,
-                "failed to determine max proof size for sector {}: {}",
-                sector_number,
-                e
-            )
-        })?;
-        if params.proof.len() > max_proof_size {
+        let proof_lengths = vec![Some(params.proof.len())];
+        let precommits = vec![precommit.clone()];
+        let (batch_return, svis) = get_valid_verify_infos(rt, &precommits, &proof_lengths)?;
+        if batch_return.success_count != 1 {
             return Err(actor_error!(
                 illegal_argument,
-                "sector prove-commit proof of size {} exceeds max size of {}",
-                params.proof.len(),
-                max_proof_size
+                "failed to get an invalid verify info from precommit"
```

Comment:
```suggestion
                "failed to get an valid verify info from precommit"
```

Or just "precommit invalid for activation"